### PR TITLE
[irods/irods#4506] Add iunreg, deprecate irm -U (4-2-stable)

### PR DESCRIPTION
**Pull request by alanking**
CI tests are running.

Requires irods/irods#4730

### CODE REVIEW

File: src/iunreg.cpp [hunk 1]
```diff
@@ -0,0 +1,162 @@
+/*** Copyright (c), The Regents of the University of California            ***
```
Comment (trel) [inline src/iunreg.cpp]: not california

File: src/iunreg.cpp [hunk 2]
```diff
@@ -0,0 +1,162 @@
...
+ *** For more information please refer to files in the COPYRIGHT directory ***/
+/*
+ * iunreg - The irods replica trimming utility
```
Comment (trel) [inline src/iunreg.cpp]: not replica trimming

**Comment by trel** [comment]
merged to master - so 4-2 can get merged if you mark as ready with a #

**Comment by alanking** [comment]
#s added. Alleviates some of the test failures, but log flushing continues to be an ongoing issue.

**Pull request merged**